precompile/contracts/feemanager/event.go

```diff
@@ -0,0 +1,68 @@
+// Code generated
+// This file is a generated precompile contract config with stubbed abstract functions.
+// The file is generated by a template. Please inspect every code and comment in this file before use.
+
+package feemanager
+
+import (
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/commontype"
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const FeeConfigChangedEventBaseGasCost = contract.LogGas + contract.LogTopicGas
+
+// ContractChangeFeeConfig represents a ChangeFeeConfig non-indexed event data raised by the Contract contract.
+type ChangeFeeConfigEventData struct {
+	GasLimit                 *big.Int
+	TargetBlockRate          *big.Int
+	MinBaseFee               *big.Int
+	TargetGas                *big.Int
+	BaseFeeChangeDenominator *big.Int
+	MinBlockGasCost          *big.Int
+	MaxBlockGasCost          *big.Int
+	BlockGasCostStep         *big.Int
+}
```

Comment:
Could we add a comment to explain why we need the intermediate type between `commontype.FeeConfig` and `ChangeFeeConfigEventData` ?

Reply:
I think we can just unexport it and return commontype.FeeConfig instead.